Cargo.toml

```diff
@@ -189,6 +189,7 @@ yatp = { workspace = true }
 [dev-dependencies]
 api_version = { workspace = true, features = ["testexport"] }
 engine_test = { workspace = true }
+example_coprocessor_plugin = { workspace = true }                              # must, used for testing coprocessor plugin
```

Comment:
Should not be marked with `useless`, as it will be used when testing the validation on coprocessor plugin